autoload/vebugger.vim

```diff
@@ -268,26 +270,39 @@ function! vebugger#createDebugger(command)
     return l:debugger
 endfunction
 
+if has('timers')
+    function! ReadingTimerCallback(debugger, timer_id)
+        call a:debugger.invokeReading()
+    endfunction
+endif
 
 "Create a debugger and set it as the currently active debugger
 function! vebugger#startDebugger(command)
     call vebugger#killDebugger()
 
     let s:debugger=vebugger#createDebugger(a:command)
 
-    augroup vebugger_shell
-        autocmd!
-        autocmd CursorHold * call s:debugger.invokeReading()
-    augroup END
+    if has('timers')
+        let s:timer_id = timer_start(500, function('ReadingTimerCallback', [s:debugger]), {'repeat': -1})
```

Comment:
Neovim does not support partial functions. Since `s:debugger` is global, just make the callback use it.